src/Search.js

```diff
@@ -12,8 +12,8 @@ function autoCompleteCity(city) {
   return fetch(`${autocompleteURL}${query}`)
     .then((res) => res.json())
     .then((result) => {
-      console.log(result);
-      if(!result.items) return []
+      // console.log(result);
```

Comment:
Instead of commenting you can remove all the console.log statement

Reply:
solved.